### PR TITLE
[stable/kuberos] add support for custom annotations in service

### PR DESCRIPTION
**Pull request by cezarsa**
#### What this PR does / why we need it:

Adds optional annotations to the service created by the stable/kuberos chart.

#### Special notes for your reviewer:

cc @timothyclarke

#### Checklist
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/kuberos/values.yaml [hunk 1]
```diff
@@ -62,6 +62,8 @@ image:
 service:
   type: ClusterIP
   port: 80
+  annotations: {}
```
Comment (cpanato) [inline stable/kuberos/values.yaml:65]: can you add this in the readme as well?

Comment (cezarsa) [reply stable/kuberos/values.yaml:65]: Done, I also added entries to the existing service fields.

**Pull request merged**